### PR TITLE
Updates for Biometrics and Genral Parse Stat

### PR DESCRIPTION
**Pull request by rhshah**

### CODE REVIEW

File: install_data.sh [hunk 1]
```diff
@@ -1,18 +1,27 @@
...
+foldername=test_data
+
+# Skip if already have test data
+[[ -f $filename ]] && exit 0
+[[ -d $foldername ]] && exit 0
+
+#curl -c ./cookie -s -k -L "https://drive.google.com/uc?export=download&id=$fileid" > /dev/null
+#curl -k -Lb ./cookie "https://drive.google.com/uc?export=download&confirm=`awk '/download/ {print $NF}' ./cookie`&id=${fileid}" -o ${filename}
+cd tests/inputs || exit
+curl -L "https://drive.usercontent.google.com/download?id=${fileid}&confirm=xxx" -o ${filename}
```
Comment (carmelinacharalambous) [inline install_data.sh:18]: curl works for me

File: nucleo_aggregate_visualize.cwl
Comment (carmelinacharalambous) [inline nucleo_aggregate_visualize.cwl:1]: getting error when pulling 0.3.2 after docker login

kcharanambou$ docker pull ghcr.io/msk-access/cci_utils:0.3.2
Error response from daemon: Head "https://ghcr.io/v2/msk-access/cci_utils/manifests/0.3.2": denied: denied

**Review by carmelinacharalambous — COMMENTED** [review]
- curl link for data works locally
- docker pull for cci_util 0.3.2 works locally
- docker pull for biometrics 0.2.15 works locally 
- nucleo_aggregate_visualize.cwl has been updated to cci_utils 0.3.2

**Review by buehlere — APPROVED** [review]
cci_utils update looks correct. Thanks for fixing folder structure.

**Review by carmelinacharalambous — APPROVED** [review]
- curl link for data works locally
- docker pull for cci_util 0.3.2 works locally
- docker pull for biometrics 0.2.15 works locally
- nucleo_aggregate_visualize.cwl has been updated to cci_utils 0.3.2

**Pull request merged**